src/app/components/MediaLoader/index.tsx

```diff
@@ -144,10 +144,16 @@ const MediaContainer = ({ playerConfig, showAds }: MediaContainerProps) => {
 type Props = {
   blocks: MediaBlock[];
   className?: string;
+  counterNameOverride?: string;
```

Comment:
This should now be ``pageIdentifierOverride` instead of countername override: see https://github.com/bbc/simorgh/pull/11782/files#r1750005785
```suggestion
  pageIdentifierOverride?: string;
```